File: MasterErp.Entities/Models/PaymentReceipt.cs
using MasterErp.Entities.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterErp.Entities.Models
{
    public class PaymentReceipt
    {
        [Key]
        public int PaymentReceiptId { get; set; }
        public int ReceiptNumber { get; set; }
        public int ReceiptLedgerId { get; set; }
        public int PaymentTypeId { get; set; }
        public int? PaidWithId { get; set; }
        public int? SafeId { get; set; }
        public int AgencyTypeId { get; set; }
        public int AgencyId { get; set; }
        public int AccountId { get; set; }
        public double MoneyAmount { get; set; }
        public bool IsLocked { get; set; }
        public bool IsCancelled { get; set; }
        public string BenefitPerson { get; set; }
        public string ChequeNumber { get; set; }
        public string DocNumber { get; set; }
        public string Notes { get; set; }
        public string InsertUser { get; set; }
        public string UpdateUser { get; set; }
        public DateTime ReleaseDate { get; set; }
        public DateTime InsertDate { get; set; }
        public DateTime? UpdateDate { get; set; }

    }
}

File: MasterErp.Angular/src/app/components/GeneralAccounts/models/GeneralAccounts/PaymentReceipt.ts
export interface PaymentReceipt {
    paymentReceiptId: number;
    receiptNumber: number;    
    paymentTypeId: number;    
    receiptLedgerId: number;
    releaseDate: string;
    contactName: string,
    bankAccountId: number | null;
    safeId: number | null;
    agencyTypeId: number;
    accountId: number | null;
    supplierId: number | null;
    customerId: number | null;
    employeeId: number | null;
    moneyAmount: number;
    docNumber: string;
    currencyId: number | null;
    description: string;
    isLocked: boolean;
    isCancelled: boolean;
    createdBy: string;
    createdDate: string;
    modifiedBy: string;
    modifiedDate: string | null;
}
export interface ReceiptLedger {
    receiptLedgerId: number;
    startReceiptNumber: number;
    periodId: number;
    receiptLedgerTypeId: number;
    operationTypeId: number;
    isActive: boolean;
    isLocked: boolean;
    code: string;
    nameAR: string;
    nameEN: string;
    notes: string;
    insertUser: string;
    updateUser: string;
    insertDate: string;
    updateDate: string | null;
}

